src/main/scala/cromwell/engine/backend/local/SharedFileSystem.scala

```diff
@@ -21,11 +20,18 @@ object SharedFileSystem {
   val SharedFileSystemConf = ConfigFactory.load.getConfig("backend").getConfig("shared-filesystem")
   val CromwellExecutionRoot = SharedFileSystemConf.getString("root")
   val LocalizationStrategies = SharedFileSystemConf.getStringList("localization").asScala
-  val Localizers = LocalizationStrategies map {
+  val Localizers = localizePathAlreadyLocalized _ +: (LocalizationStrategies map {
     case "hard-link" => localizePathViaHardLink _
     case "soft-link" => localizePathViaSymbolicLink _
     case "copy" => localizePathViaCopy _
     case unsupported => throw new UnsupportedOperationException(s"Localization strategy $unsupported is not recognized")
+  })
+
+  /**
+   * Return a `Success` result if the file has already been localized, otherwise `Failure`.
+   */
+  private def localizePathAlreadyLocalized(call: Option[Call], originalPath: Path, executionPath: Path): Try[Unit] = {
+    if (Files.exists(executionPath)) Success(()) else Failure(new Throwable)
```

Comment:
I made a local version of Cromwell that surfaced errors with a `recoverWith`, but I wasn't too happy with that since localizer failure is somewhat expected and I ended up with lots of scary but harmless errors.  

We should be able to run a `Validation` over the `Stream` of localizers if *all* of them fail, though even in the pathological case George ran into that wasn't the case.